src/meta/src/backup_restore/restore.rs

```diff
@@ -218,280 +192,265 @@ pub async fn restore(opts: RestoreOpts) -> BackupResult<()> {
-        restore_impl(opts.clone(), None, Some(backup_store.clone()))
-            .await
-            .unwrap();
-    }
-
-    #[tokio::test]
-    #[should_panic]
-    async fn test_sanity_check_monotonicity_requirement() {
-        let opts = get_restore_opts();
-        let backup_store = get_backup_store(opts.clone()).await.unwrap();
-        let snapshot = MetaSnapshot {
-            id: opts.meta_snapshot_id,
-            metadata: ClusterMetadata {
-                default_cf: HashMap::from([(vec![1u8, 2u8], memcomparable::to_vec(&10).unwrap())]),
-                system_param: get_system_params(),
-                ..Default::default()
-            },
-            ..Default::default()
-        };
-        backup_store.create(&snapshot, None).await.unwrap();
-
-        // violate monotonicity requirement
-        let mut snapshot_2 = MetaSnapshot {
-            id: snapshot.id + 1,
-            ..Default::default()
-        };
-        snapshot_2
-            .metadata
-            .default_cf
-            .insert(vec![1u8, 2u8], memcomparable::to_vec(&9).unwrap());
-        backup_store.create(&snapshot_2, None).await.unwrap();
-        restore_impl(opts.clone(), None, Some(backup_store.clone()))
-            .await
-            .unwrap();
-    }
-
-    #[tokio::test]
-    async fn test_dry_run() {
-        let mut opts = get_restore_opts();
-        assert!(!opts.dry_run);
-        opts.dry_run = true;
-        let backup_store = get_backup_store(opts.clone()).await.unwrap();
-        let empty_meta_store = get_meta_store(opts.clone()).await.unwrap();
-        let system_param = get_system_params();
-        let snapshot = MetaSnapshot {
-            id: opts.meta_snapshot_id,
-            metadata: ClusterMetadata {
-                default_cf: HashMap::from([
-                    (
-                        "some_key_1".as_bytes().to_vec(),
-                        memcomparable::to_vec(&10).unwrap(),
-                    ),
-                    (
-                        "some_key_2".as_bytes().to_vec(),
-                        memcomparable::to_vec(&"some_value_2".to_string()).unwrap(),
-                    ),
-                ]),
-                hummock_version: {
-                    let mut version = HummockVersion::default();
-                    version.id = HummockVersionId::new(123);
-                    version
-                },
-                system_param: system_param.clone(),
-                ..Default::default()
-            },
-            ..Default::default()
-        };
-        backup_store.create(&snapshot, None).await.unwrap();
-        restore_impl(
-            opts.clone(),
-            Some(empty_meta_store.clone()),
-            Some(backup_store.clone()),
-        )
-        .await
-        .unwrap();
-
-        dispatch_meta_store!(empty_meta_store, store, {
-            assert!(SystemParams::get(&store).await.unwrap().is_none());
-        });
-    }
-}
+// #[cfg(test)]
+// mod tests {
```

Comment:
Cc @zwang28 , I commented out all unit tests about backup and restore, which is using `MetaSnapshotV1`. Please switch it to use sql backend after this PR get merged.